redirects-transition.conf

```diff
@@ -4,9 +4,34 @@
 rewrite ^/about.shtml https://www.fec.gov/about/ redirect;
 
 # about/offices/ redirects
+rewrite ^/about/offices/adm/admin.shtml /about/leadership-and-structure/fec-offices/office-of-the-staff-director/ redirect;
+rewrite ^/about/offices/adr/adr.shtml /about/leadership-and-structure/fec-offices/compliance-offices/ redirect;
+rewrite ^/about/offices/audit/audit.shtml /about/leadership-and-structure/fec-offices/compliance-offices/ redirect;
+rewrite ^/about/offices/budget/budget.shtml /about/leadership-and-structure/fec-offices/office-of-the-chief-financial-officer/ redirect;
+rewrite ^/about/offices/CCO/CCO.shtml /about/leadership-and-structure/fec-offices/office-of-the-staff-director/ redirect;
+rewrite ^/about/offices/CIO/CIO.shtml /about/leadership-and-structure/fec-offices/office-of-the-chief-information-officer/ redirect;
+rewrite ^/about/offices/CFO/CFO.shtml /about/leadership-and-structure/fec-offices/office-of-the-chief-financial-officer/ redirect;
+rewrite ^/about/offices/CDEAO/CDEAO.shtml /about/leadership-and-structure/fec-offices/office-of-the-staff-director/ redirect;
+rewrite ^/about/offices/cong_affairs/cong_affairs.shtml /about/leadership-and-structure/fec-offices/communication-offices/ redirect;
+rewrite ^/about/offices/DSDMA/DSDMA.shtml /about/leadership-and-structure/fec-offices/management-and-administration-offices/ redirect;
+rewrite ^/about/offices/EEO/EEO.shtml https://www.fec.gov/about/equal-employment-opportunity/ redirect;
+rewrite ^/about/offices/finance/finance.shtml /about/leadership-and-structure/fec-offices/office-of-the-chief-financial-officer/ redirect;
+rewrite ^/about/offices/finance/procurement.shtml /about/leadership-and-structure/fec-offices/office-of-the-chief-financial-officer/ redirect;
+rewrite ^/about/offices/hr/hr.shtml /about/leadership-and-structure/fec-offices/management-and-administration-offices/ redirect;
+rewrite ^/about/offices/info/info.shtml /about/leadership-and-structure/fec-offices/communication-offices/ redirect;
+rewrite ^/about/offices/oar/oar.shtml /about/leadership-and-structure/fec-offices/compliance-offices/ redirect;
 rewrite ^/about/offices/offices.shtml https://www.fec.gov/about/leadership-and-structure/fec-offices/ redirect;
+rewrite ^/about/offices/OGC/OGC.shtml /about/leadership-and-structure/fec-offices/office-of-general-counsel/ redirect;
+rewrite ^/about/offices/OGC/DepGC.shtml /about/leadership-and-structure/fec-offices/office-of-general-counsel/ redirect;
```

Comment:
Example of redirecting to a named [anchor](https://www.fec.gov/about/leadership-and-structure/fec-offices/office-of-general-counsel/#deputy-general-counseladministration):

```rewrite ^/about/offices/OGC/DepGC.shtml https://www.fec.gov/about/leadership-and-structure/fec-offices/office-of-general-counsel/#deputy-general-counseladministration redirect;```

Reply:
@patphongs Great idea. I've put them in where applicable and have updated the redirect spreadsheet. It's ready for your review.